usaon_benefit_tool/routes/root.py

```diff
@@ -1,8 +1,10 @@
 from flask import Blueprint, render_template
 
+from usaon_benefit_tool.constants import repo
+
 root_bp = Blueprint('root', __name__, url_prefix='/')
 
 
 @root_bp.route('')
 def root():
-    return render_template('home.html')
+    return render_template('home.html', url=repo.REPO_URL)
```

Comment:
Instead of passing in `REPO_URL` to each call to `render_template` separately, could you add `repo_url=REPO_URL`  as a global here? https://github.com/nsidc/usaon-benefit-tool/blob/main/usaon_benefit_tool/__init__.py#L126-L129

Then we don't need to pass it in, we can just use it in a template :)